.size-limit.js

```diff
@@ -94,7 +94,7 @@ module.exports = [
     path: 'packages/browser/build/bundles/bundle.tracing.min.js',
     gzip: false,
     brotli: false,
-    limit: '100 KB',
+    limit: '105 KB',
```

Comment:
side note (no action required): I really hope that our bundle size will decrease to ~v7.0.0 levels once we removed all the deprecated stuff

Reply:
yeah, it will decrease for sure - right now we need to add a certain degree of duplication, sadly, which will go away again! I'm very optimistic in this regard.